libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/NavigationOptionsEx.kt

```diff
@@ -0,0 +1,11 @@
+@file:JvmName("NavigationOptionsExtensions")
+
+package com.mapbox.navigation.base.internal.extensions
+
+import com.mapbox.navigation.base.internal.CopilotOptions
+import com.mapbox.navigation.base.options.NavigationOptions
+
+fun NavigationOptions.Builder.copilotOptions(copilotOptions: CopilotOptions) =
+    this.copilotOptions(copilotOptions).build()
```

Comment:
I think we should return `NavigationOptions.Builder`, so that we are not forced to called it last. 
```suggestion
    this.copilotOptions(copilotOptions)
```

Reply:
Great point! Fixing.

Reply:
Fixed 👍